Fix errorprone warnings in CloudLibraryProjectStateTest.java

google-cloud-intellij/google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/apis/CloudLibraryProjectStateTest.java:119: warning: [CatchFail] Ignoring exceptions and calling fail() is unnecessary, and makes test output less useful
                        try {
                        ^
    (see http://errorprone.info/bugpattern/CatchFail)
google-cloud-intellij/google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/apis/CloudLibraryProjectStateTest.java:120: warning: [DefaultCharset] Implicit use of the platform default charset, which can result in e.g. non-ASCII characters being silently replaced with '?' in many environments
                          mavenProject.getFile().setBinaryContent("not a valid pom.xml".getBytes());
                                                                                                ^
    (see http://errorprone.info/bugpattern/DefaultCharset)
  Did you mean 'mavenProject.getFile().setBinaryContent("not a valid pom.xml".getBytes(UTF_8));' or 'mavenProject.getFile().setBinaryContent("not a valid pom.xml".getBytes(Charset.defaultCharset()));'?

--- a/google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/apis/CloudLibraryProjectStateTest.java
+++ b/google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/apis/CloudLibraryProjectStateTest.java
@@ -122,7 +122,7 @@ public class CloudLibraryProjectStateTest {
                               .setBinaryContent(
                                   "not a valid pom.xml".getBytes(Charset.defaultCharset()));
                         } catch (IOException e) {
-                          throw new AssertionError(e);
+                          throw new AssertionError("failed to write test content to pom.xml", e);
                         }
                       });
 

--- a/google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/apis/CloudLibraryProjectStateTest.java
+++ b/google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/apis/CloudLibraryProjectStateTest.java
@@ -17,7 +17,6 @@
 package com.google.cloud.tools.intellij.apis;
 
 import static com.google.common.truth.Truth.assertThat;
-import static org.junit.Assert.fail;
 import static org.mockito.Mockito.when;
 
 import com.google.cloud.tools.intellij.MavenTestUtils;
@@ -36,6 +35,7 @@ import com.intellij.openapi.module.Module;
 import com.intellij.testFramework.fixtures.IdeaProjectTestFixture;
 import com.intellij.util.xml.DomUtil;
 import java.io.IOException;
+import java.nio.charset.Charset;
 import java.util.List;
 import java.util.Set;
 import org.jetbrains.annotations.NotNull;
@@ -117,9 +117,12 @@ public class CloudLibraryProjectStateTest {
                   .runWriteAction(
                       () -> {
                         try {
-                          mavenProject.getFile().setBinaryContent("not a valid pom.xml".getBytes());
+                          mavenProject
+                              .getFile()
+                              .setBinaryContent(
+                                  "not a valid pom.xml".getBytes(Charset.defaultCharset()));
                         } catch (IOException e) {
-                          fail();
+                          throw new AssertionError(e);
                         }
                       });
 

